susemanager-utils/susemanager-sls/src/modules/sumautil.py

```diff
@@ -127,32 +127,33 @@ def get_kernel_live_version():
 
         salt '*' sumautil.get_kernel_live_version
     '''
-    kernel_live_version = _kgr()
+    kernel_live_version = _klp()
     if not kernel_live_version:
         log.debug("No kernel live patch is active")
 
     return kernel_live_version
 
-def _kgr():
+def _klp():
     '''
-    kgr to identify the current kgraft patch
+    klp to identify the current kernel live patch
 
     :return:
     '''
-    kgr = salt.utils.which_bin(['kgr'])
+    # get 'kgr' for versions prior to SLE 15
+    klp = salt.utils.which_bin(['klp', 'kgr'])
```

Comment:
Seems salt moved which_bin:

```
    salt.utils.versions.warn_until(
        'Neon',
        'Use of \'salt.utils.which_bin\' detected. This function has been '
        'moved to \'salt.utils.path.which_bin\' as of Salt 2018.3.0. This '
        'warning will be removed in Salt Neon.',
        stacklevel=3
    )
```

Live patching is not supported on SLE11, right? Current salt version on SLE12 is 2018.3.0 so I think we can use the new path already.

Reply:
Good point. Updated now.